setup.py

```diff
@@ -11,10 +11,14 @@
 README_PATH = os.path.join(BASE, "README.rst")
 
 install_requires = [
+    'aiohttp',
     'aiohttp-transmute',
     'aiohttp_cors',
     'docopt',
     'jinja2',
+    'aiozipkin',
+    'aiotask_context',
+    'orbital-core'
```

Comment:
i don't believe we need to self reference this lib.

should we add these new packages in alphabetical order?

Reply:
yes, let's not self-reference... that doesn't make a lot of sense. +1 on alphabetical but nice to have IMO.

Reply:
Must have added it mistakenly. Removed it now.